andela/wg-aces
# #159790003 Add Surplus Migration Files To Git Repo

#### What does this PR do?
Adds surplus migration files

#### Description of Task to be completed?
This task is meant to include all the migration files that are initially created when the develop branch is pulled and  the command `./manage.py makemigrations` is run.

#### How should this be manually tested?
After cloning this repository;
    -  switch into project directory by running `cd wg-aces`
    -  switch to chore branch by running `git checkout ch-add-surplus-migrations-git-159790003`
    -  Run the command `./manage.py makemigrations` in your terminal

You should see the result: *No changes detected*

#### What are the relevant pivotal tracker stories?
[#159790003](https://www.pivotaltracker.com/story/show/159790003)

#### Screenshots
N/A
